crates/services/upgradable-executor/src/executor.rs

```diff
@@ -617,7 +660,9 @@ where
         mode: ProduceBlockMode,
         new_tx_waiter: impl NewTxWaiterPort,
         preconfirmation_sender: impl PreconfirmationSenderPort,
-    ) -> ExecutorResult<Uncommitted<ExecutionResult, Changes>>
+    ) -> ExecutorResult<
+        Uncommitted<(ExecutionResult, Vec<StorageReadReplayEvent>), Changes>,
```

Comment:
Would be nice to have some alias, because it is a huge type now=D

Reply:
Sure. Done in 1deb59dfbe.